src/program.js

```diff
@@ -210,6 +210,11 @@ Example: $0 --help run.
           demand: true,
           type: 'string',
         },
+        'api-proxy': {
+          describe: 'Use a proxy to access the signing API',
```

Comment:
I would add another sentence to this: "Example: https://yourproxy:6000"

This will help people understand what kind of value they would need to pass to this option.